include/boost/dynamic_bitset/dynamic_bitset.hpp

```diff
@@ -1622,6 +1624,17 @@ inline bool operator>=(const dynamic_bitset<Block, Allocator>& a,
     return !(a < b);
 }
 
+//-----------------------------------------------------------------------------
+// hash operations
+
+template <typename Block, typename Allocator>
+inline std::size_t hash_value(const dynamic_bitset<Block, Allocator>& a)
+{
+    std::size_t res = hash_value(a.m_num_bits);
+    hash_combine(res, hash_value(a.m_bits));
```

Comment:
This does not look right to me. Why are you calling hash_value here?

Reply:
Thanks, fixed in the next push.